_reports/2023/opening.md

```diff
@@ -0,0 +1,101 @@
+organization, Brink always puts review and testing first and foremost,
+and we've tried to reflect that part of our internal culture in this
+external report.
+
+The end of this report also includes financial information about Brink,
+information about the work performed by our executive director and our
+operations manager during 2023, and details about the make up of our
+board and our grant committee---everything you'd typically expect to
+find in a charitable organization's annual report.  But Brink's primary
+mission is to find amazing Bitcoin developers and fund them on behalf of
+our sponsors, and we couldn't be more excited to start this report by
+showcasing all that they've accomplished in 2023.
+
+## 2023 developer highlights
+
+_These are the highlights.  For details about each developer's
+work, see later in this report._
+
+Sebastian Falbesoner
+: left over 300 review comments as a part-time engineer, many of them
+  focused on version 2 encrypted peer-to-peer transport.  He also began
+  contributing the the libsecp256k1 cryptographic library used by Bitcoin
+  Core and several other security-focused projects.
+
+Michael Ford
+: continued his role as project maintainer, leaving almost 1,600
+  review comments, merging an average of 11 pull requests a week, and
+  releasing every 2023 version of Bitcoin Core.  He helped lead several
+  efforts to modernize Bitcoin Core's build toolchain.
+
+Niklas Gögge
+: left over 500 review comments, publicized several responsible disclosures
+  he had made of serious vulnerabilities, found multiple new bugs in
+  Bitcoin Core pull requests before they were merged, significantly
+  extended Bitcoin Core's testing infrastructure, and made several
+  safety-focused changes to Bitcoin Core's code.
+
+Fabian Jahr
+: left over 200 review comments after joining Brink mid-2023, led major
+  improvements to distributed generation of ASMap files, wrote
+  proof-of-concept code for batch validation of schnorr signatures,
+  researched cross-input signature aggregation, and added significant
+  resiliency to Bitcoin Core's code hosting.
+
+Hennadii Stepanov
+: left almost 1,300 review comments, contributing greatly to the
+  project's effort to modernize its build tool chain, and continued his
+  role as the project's GUI maintainer.  He also continued working with the
+  Bitcoin Design Community on a reference GUI wallet design compatible
+  with Bitcoin Core.
+
+Stéphan Vuylsteke
+: left almost 600 review comments, earning special commendation from his
+  peers for his diligence and follow up.  He continued contributing to multiple
+  education and mentorship efforts, including Qala, London BitDevs, and
+  the Bitcoin Core Pull Request Review Club.
+
+Gloria Zhao
+: continued her role as P2P system maintainer, leaving 900 review
+  comments and leading the work on package relay.  She was also a
+  leading contributor to TRUC (v3) transactions and cluster mempool.  She
```

Comment:
Ancestor-aware funding might be a better example, as I definitely wrote MiniMiner (n27021) in 2023, but not sure where cluster mempool was in 2023. I have been doing a lot of cluster mempool review this year... but in 2023, it was mostly Pieter and Suhas doing research iirc.

```suggestion
  leading contributor to TRUC (v3) transactions and ancestor-aware funding.  She
```